ingenialink/ethercat/register.py

```diff
@@ -0,0 +1,76 @@
+from typing import Optional, Any, Union, Tuple, Dict, List
+
+from ingenialink.canopen.register import CanopenRegister
+from ingenialink.enums.register import REG_DTYPE, REG_ACCESS, REG_PHY, REG_ADDRESS_TYPE
+
+
+class EthercatRegister(CanopenRegister):
+    """EtherCAT Register.
+
+    Args:
+        idx: Index of the register.
+        subidx: Subindex of the register.
+        dtype: Data type.
+        access: Access type.
+        identifier: Identifier.
+        units: Units.
+        cyclic: Cyclic typed register.
+        phy: Physical units.
+        subnode: Subnode.
+        storage: Storage.
+        reg_range: Range (min, max).
+        labels: Register labels.
+        enums: Enumeration registers.
+        cat_id: Category ID.
+        scat_id: Sub-category ID.
+        internal_use: Internal use.
+        address_type: Address Type.
+
+    Raises:
+        TypeError: If any of the parameters has invalid type.
+        ILValueError: If the register is invalid.
+        ILAccessError: Register with wrong access type.
+
+    """
+
+    def __init__(
```

Comment:
You can delete the whole constructor and just put a `pass` instead.
In addition, as the constructor is the same, you can delete the `Args` and `Raises` fields in docstrings.

Reply:
Done